crypto/crypto/src/unit_tests/x25519_test.rs

```diff
@@ -1,76 +0,0 @@
-// Copyright (c) The Libra Core Contributors
-// SPDX-License-Identifier: Apache-2.0
-
-use crate::{traits::*, x25519::*};
-use rand::{rngs::StdRng, SeedableRng};
-
-#[test]
```

Comment:
not a single test to actually ensure an exchange works? or how to perform an exchange?

Reply:
it's all in dalek :)

Reply:
oh this was not a knock on this PR, but the test that was here.... it is concerning.